examples/deployment/algob.config.js

```diff
@@ -58,9 +58,9 @@ let accounts = mkAccounts([
 // accounts = accounts.concat(accFromFile);
 
 let defaultCfg = {
-  host: "http://localhost",
-  port: 46469,
-  token: "aade468d25a7aa48fec8082d6a847c48492066a2741f3731e613fdde086cd6e9",
+  host: "127.0.0.1",
+  port: 43103,
+  token: "11bca60d574ac896e2efbd454a2e7a06ab540002e14975f9f0e56abd61d8f5ab",
```

Comment:
Don't change this too.
I'm not sure how we can have this different per-developer and still maintain all demo features.

Reply:
OK